src/test/resources/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs/multiSamples.GATK3expected.g.vcf

```diff
@@ -137,8 +137,8 @@
-20	899942	.	G	A	4913.79	.	AC=19;AF=0.792;AN=24;AS_BaseQRankSum=0.400;AS_FS=0.000;AS_InbreedingCoeff=0.5567;AS_MQ=44.39;AS_MQRankSum=-0.300;AS_QD=30.62;AS_ReadPosRankSum=-1.500;AS_SOR=1.893;BaseQRankSum=0.493;ClippingRankSum=0.00;DP=148;ExcessHet=0.0000;FS=0.000;InbreedingCoeff=0.4850;MLEAC=30;MLEAF=1.00;MQ=43.13;MQRankSum=-2.530e-01;QD=28.17;RAW_MQ=264157.00;ReadPosRankSum=-1.501e+00;SOR=1.923	GT:AD:DP:GQ:PGT:PID:PL	1/1:0,3:3:9:1|1:899928_G_C:135,9,0	1/1:0,15:15:45:1|1:899928_G_C:636,45,0	1/1:0,16:16:45:1|1:899928_G_C:636,45,0	./.:0,0:0:0:.:.:0,0,0	./.:0,0:0:0:.:.:0,0,0	./.:0,0	./.:1,0:1:0:.:.:0,0,0	1/1:0,9:9:27:1|1:899928_G_C:366,27,0	./.:3,0:3:0:.:.:0,0,0	./.:0,0:0:0:.:.:0,0,0	./.:2,0:2:0:.:.:0,0,0	./.:1,0:1:0:.:.:0,0,0	1/1:0,7:7:19:1|1:899928_G_C:273,19,0	./.:0,0	0/1:2,8:10:88:0|1:899928_G_C:88,0,274	./.:0,0:0:0:.:.:0,0,0	1/1:0,58:58:99:1|1:899928_G_C:2217,159,0	1/1:0,5:5:13:1|1:899928_G_C:183,13,0	./.:2,0:2:0:.:.:0,0,0	0/0:5,0:5:15:.:.:0,15,178	1/1:0,7:7:16:1|1:899928_G_C:189,16,0	1/1:0,2:2:6:1|1:899928_G_C:90,6,0	./.:0,0:0:0:.:.:0,0,0	0/0:1,0:1:3:.:.:0,3,36
+20	899928	.	G	C	4944.58	.	AC=21;AF=0.808;AN=26;AS_BaseQRankSum=-1.400;AS_FS=0.000;AS_InbreedingCoeff=0.5339;AS_MQ=43.54;AS_MQRankSum=-0.700;AS_QD=25.36;AS_ReadPosRankSum=0.000;AS_SOR=1.863;BaseQRankSum=-1.383e+00;ClippingRankSum=0.00;DP=133;ExcessHet=0.0000;FS=0.000;InbreedingCoeff=0.5089;MLEAC=25;MLEAF=0.962;MQ=42.34;MQRankSum=-6.740e-01;QD=28.73;RAW_MQ=222270.00;ReadPosRankSum=0.00;SOR=1.781	GT:AD:DP:GQ:PGT:PID:PL	1/1:0,3:3:9:1|1:899928_G_C:135,9,0	1/1:0,18:18:54:1|1:899928_G_C:767,54,0	1/1:0,15:15:45:1|1:899928_G_C:640,45,0	./.	./.	./.	./.	1/1:0,9:9:27:1|1:899928_G_C:370,27,0	1/1:0,1:1:3:1|1:899928_G_C:45,3,0	./.:1,0:1:0:.:.:0,0,0	./.:2,0:2:0:.:.:0,0,0	./.:3,0:3:0:.:.:0,0,0	1/1:0,5:5:15:1|1:899928_G_C:225,15,0	./.	0/1:2,2:4:78:0|1:899928_G_C:78,0,282	./.	1/1:0,51:51:99:1|1:899928_G_C:2229,153,0	1/1:0,3:3:9:1|1:899928_G_C:135,9,0	./.:2,0:2:0:.:.:0,0,0	0/0:7,0:7:21:.:.:0,21,236	1/1:0,4:4:12:1|1:899928_G_C:168,12,0	1/1:0,2:2:6:1|1:899928_G_C:90,6,0	./.	0/0:2,0:2:6:.:.:0,6,65
+20	899933	.	GGGGGGCGC	G	30.86	.	AC=2;AF=0.063;AN=32;AS_BaseQRankSum=.;AS_FS=0.000;AS_InbreedingCoeff=0.2616;AS_MQ=60.00;AS_MQRankSum=.;AS_QD=30.97;AS_ReadPosRankSum=.;AS_SOR=0.693;DP=128;ExcessHet=0.0000;FS=0.000;InbreedingCoeff=0.3190;MLEAC=1;MLEAF=0.031;MQ=34.64;QD=30.86;RAW_MQ=3600.00;SOR=1.609	GT:AD:DP:GQ:PGT:PID:PL	0/0:3,0:3:9:.:.:0,9,97	0/0:18,0:18:27:.:.:0,27,405	0/0:15,0:15:24:.:.:0,24,360	./.	./.	./.	./.	0/0:8,0:8:21:.:.:0,21,315	1/1:0,1:1:3:1|1:899928_G_C:45,3,0	0/0:1,0:1:3:.:.:0,3,33	./.:2,0:2:0:.:.:0,0,0	0/0:1,0:1:3:.:.:0,3,34	0/0:6,0:6:15:.:.:0,15,225	./.	0/0:4,0:4:9:.:.:0,9,135	./.	0/0:52,0:52:99:.:.:0,120,1800	0/0:3,0:3:9:.:.:0,9,115	0/0:2,0:2:3:.:.:0,3,45	0/0:6,0:6:15:.:.:0,15,225	0/0:3,0:3:6:.:.:0,6,90	0/0:2,0:2:6:.:.:0,6,77	./.	0/0:1,0:1:3:.:.:0,3,36
```

Comment:
Annotations like InbreedingCoeff (and sometimes ExcessHet) may also change once we get rid of 0,0,0 PLs